tests/cpu/CMakeLists.txt

```diff
@@ -239,7 +239,8 @@ set(TESTS
 	transforms/Lut3DTransform_tests.cpp
 	transforms/MatrixTransform_tests.cpp
 	transforms/RangeTransform_tests.cpp
-	UnitTestLogUtils.cpp
+    UnitTestLogUtils.cpp
+    UnitTestPathUtils.cpp
```

Comment:
The indentation in all `CMakeLists.txt` files (\t) is not consistent with the code. (4 whitespaces) Is it intensional?

Reply:
Yes, some CMakeLists.txt are not consistent with the coding which mentions 4 whitespaces and no tabulation so you can fix that.

Reply:
Ok sir. Fixed it.